src/adapters/places.ts

```diff
@@ -0,0 +1,85 @@
+import { AppComponents, PlaceNotFoundError } from '../types'
```

Comment:
Just TOC. Why do we name places in plural while land and world are singular?

Reply:
because places is the service

Reply:
let's go with pluras